app/javascript/controllers/file_auth_controller.js

```diff
@@ -266,4 +270,12 @@ export default class extends Controller {
     return json.status == '401' && 'heading' in json && 'en' in json.heading && json.heading.en.length
       && json.heading.en[0].startsWith('Access is restricted to the')
   }
+
+  // Checks the result of the probe auth request to see if access is restricted to location
+  // This code depends on the text returned by probe service, so changes to the heading
+  // should be reflected here as well.
+  isNoAccess(json) {
+    return json.status == '401' && 'heading' in json && 'en' in json.heading && json.heading.en.length
```

Comment:
If we get https://github.com/sul-dlss/stacks/pull/1405 and https://github.com/sul-dlss/sul-embed/pull/2805 in this will be much easier and already done. Also noAccess below will not be necessary

Reply:
Yes, I noticed your work in stacks.